index.js

```diff
@@ -1,13 +1,12 @@
 const fs = require('fs')
-
 const babel = require('@babel/core')
 const inlinePlugin = require('babel-plugin-transform-inline-environment-variables')
 
-async function inlineEnv(path, verbose = false) {
+async function inlineEnv(path, options = {}, verbose = false) {
   console.log('inlining', path)
 
   const transformed = await babel.transformFileAsync(path, {
-    plugins: [babel.createConfigItem(inlinePlugin)],
+    plugins: [babel.createConfigItem([inlinePlugin, options])],
```

Comment:
I'm not sure how the plugin behaves, but if you supply an empty `include` array will it include nothing or will it just include everything?

Explicitly destructuring the options, and adding the relevant properties might be necessary!

Reply:
According to my local tests:

- if `include` is an empty array it means none of variables would be transformed
- if `include` is null it will transform all variables

Reply:
@dac09 if no more concerns can we continue to merge?

Reply:
Sounds good :)

Reply:
@bencao approved :) Not sure if you wanted me to merge or not, I'll leave it to you since you probably want to publish it too!

Reply:
thanks, I'll merge and publish a new version.